src/main/scala/viper/gobra/util/Constants.scala

```diff
@@ -0,0 +1,13 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2011-2020 ETH Zurich.
+
+package viper.gobra.util
+
+object Constants {
+
+  val ADDRESSABILITY_MODIFIER = "@"
+  val SHARE_PARAMETER_KEYWORD = "share"
```

Comment:
I'm surprise to see `share` here, I expected `shared`. If it is indeed `share` then ignore my previous comment about `SHARE_PARAMETER_KEYWORD`

Reply:
I theory we could have general `share` and `unshare` statements later.